kuibit/grid_data.py

```diff
@@ -633,7 +633,6 @@ def evaluate_with_spline(self, x, ext=2, piecewise_constant=False):
         if piecewise_constant or (
             self.num_dimensions != self.num_extended_dimensions
         ):
-
```

Comment:
I think that this change is not needed. Could you please force-push and remove it?